completion_aggregator/api/v1/views.py

```diff
@@ -371,7 +371,7 @@ def get(self, request, course_key):
 
         if 'user_ids' in request.query_params:
             user_ids = (int(id) for id in request.query_params['user_ids'].split(','))
-            enrollments = enrollments.filter(user_id__in=user_ids)
+            enrollments = enrollments.filter(user_id__in=user_ids).select_related('user')
```

Comment:
This select_related looks like it should be applied even if the `user_ids` parameter is not provided.  Please move this up to L355 and L370 (or apply it once before L372.

Reply:
@jcdyer Would L376 work?

Reply:
Going with that.